### ISSUE TITLE
in detr's config, the cls_cost's weight in 'HungarianAssigner' is 0, why?

### ISSUE THREAD

**Issue by smarengin** [liming-ai/AlignDet#7]
I have heard you: the Hungarian Algorithm needing explicit class information to achieve one-one matching, and the performance is poor if there is no pseudo-semantic labels. so I think the cls_cost is important, but you set it 0, why?

**Comment by smarengin** [comment]
Thanks you for your kindly reply.

**Comment by smarengin** [comment]
> > I have heard you: the Hungarian Algorithm needing explicit class information to achieve one-one matching, and the performance is poor if there is no pseudo-semantic labels. so I think the cls_cost is important, but you set it 0, why?
> 
> Thank you again, it is a bug when I clean the code. You can see the [pre-training config that has been released](https://huggingface.co/spaces/limingcv/AlignDet/blob/main/pretrain/selfsup_detr_cluster-id-as-class_contrastive/detr.py), it is 1.0. I will update related code

In the [config you have released](https://huggingface.co/spaces/limingcv/AlignDet/blob/main/pretrain/selfsup_detr_cluster-id-as-class_contrastive/detr.py), why there is no 'loss_contrastive'？
